### PR TITLE
[MIRROR] You can now see the food in the microwave [MDB IGNORE]

### PR DESCRIPTION
**Pull request by SkyratBot**
Original PR: https://github.com/tgstation/tgstation/pull/70203
--------------------

https://user-images.githubusercontent.com/35135081/192945375-e45cb351-afa3-4085-abf9-b90ed5dee166.mp4

## Why It's Good For The Game

he forgot

## Changelog

:cl:
qol: You can now see food inside the microwave.
/:cl:

### CODE REVIEW

**Comment by Tastyfish** [comment]
Is a spriter working on updating our custom microwave sprites for this?

**Comment by ghost** [comment]
> Is a spriter working on updating our custom microwave sprites for this?

~Juuuuust about done, expect either a secondary PR or something merged into this one~
https://github.com/Skyrat-SS13/Skyrat-tg/pull/16678

**Comment by Tastyfish** [comment]
This good?

**Comment by ghost** [comment]
> This good?

~Just about to launch a test~

Test MOSTLY succeeded. Items need a 3 to 4 px offset to the left so they display in the microwave properly but it's not super critical

File: code/modules/food_and_drinks/machinery/microwave.dm [hunk 1]
```diff
@@ -91,23 +96,92 @@
...
+/obj/machinery/microwave/update_overlays()
+	// When this is the nth ingredient, whats its pixel_x?
+	var/static/list/ingredient_shifts = list(
+		-3,
+		0,
+		-6,
+		1,
+		-7,
+		-1,
+		-5,
```
Comment (ghost) [inline code/modules/food_and_drinks/machinery/microwave.dm:111]: Don't forget to SKYRAT EDIT comment these, this is a TG file we're changing from the original :b

Though I wonder if we can re-define list/ingredient_shifts in a future file or not. IDK how lists work, or if it'd apply before or after the offsets are actually applied...

Comment (Tastyfish) [reply code/modules/food_and_drinks/machinery/microwave.dm:111]: what I get for adding a last second change at 5am, hah

**Pull request merged**